changelogs/fragments/772-Encode-files-recursively-and-test-case-for-keep-behavior.yml

```diff
@@ -0,0 +1,6 @@
+bugfixes:
+  - zos_copy - Zos_copy did not encode inner content of a inner folder on 
+    the source once is copy to the destiny.
+    Fix now encodes all content in a source folder include inner files of
```

Comment:
Here I'd say that you could change it to `Fix now encodes all content in a source directory, including subdirectories.`